/home/data/nbc/misc-projects/Salo_PowerReplication/code/conda_env/lib/python3.7/site-packages/numpy/lib/function_base.py:2642: RuntimeWarning: invalid value encountered in true_divide
  c /= stddev[:, None]
/home/data/nbc/misc-projects/Salo_PowerReplication/code/conda_env/lib/python3.7/site-packages/numpy/lib/function_base.py:2643: RuntimeWarning: invalid value encountered in true_divide
  c /= stddev[None, :]
Traceback (most recent call last):
  File "/home/data/nbc/misc-projects/Salo_PowerReplication/code/experiment-2/05_ddmra.py", line 147, in <module>
    project_dir, participants_file, target_file_pattern, confounds_pattern
  File "/home/data/nbc/misc-projects/Salo_PowerReplication/code/experiment-2/05_ddmra.py", line 100, in run_ddmra_analyses
    analyses=("scrubbing",),
  File "/home/data/nbc/misc-projects/Salo_PowerReplication/ddmra/ddmra/workflows.py", line 238, in run_analyses
    f"{smoothing_curve_distances} != {scrub_smoothing_curve_distances}"
AssertionError: [ 19.313  19.339  19.391 ... 142.976 143.01  143.059] != []
